hackathon_contributions_new_data_format/Isensee_JCB2018/visualization/getDataToBePlotted.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+
+def getDataToBePlotted(visualization_specification, measurement_data, conditionIds, i, clmn_name_unique):
```

Comment:
Use snakecase for function names (see https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)